Apps/Playground/UWP/App.cpp

```diff
@@ -213,26 +239,25 @@ void App::OnPointerMoved(CoreWindow^, PointerEventArgs^ args)
 {
     if (m_nativeInput != nullptr)
     {
-        const auto& point = args->CurrentPoint;
-        const auto& position = point->RawPosition;
-
-        m_nativeInput->MouseMove(static_cast<int>(position.X), static_cast<int>(position.Y));
-
-        /*
-         * Note: Because PointerPressed only first for the first button press (and no additional ones)
-         * and PointerReleased only fires when all buttons have been released, we need an alternative way
-         * to track additional button presses.
-         * (See Definition - PointerPressed: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerpressed?view=winrt-22000)
-         * (See Remarks - PointerReleased: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corewindow.pointerreleased?view=winrt-22000)
-         *
-         * In both of the below functions, we have three booleans that track each button's state and then we
-         * run these two functions to update any buttons that we may have missed.
-         */
-
-        if (point->IsInContact)
+        const auto& position = args->CurrentPoint->RawPosition;
+        const auto& deviceType = args->CurrentPoint->PointerDevice->PointerDeviceType;
+        const auto& deviceSlot = args->CurrentPoint->PointerId;
+        const auto& updateKind = args->CurrentPoint->Properties->PointerUpdateKind;
+        const auto& x = static_cast<int>(position.X);
+        const auto& y = static_cast<int>(position.Y);
```

Comment:
Kind of a nit, but a const ref to an int seems weird since the ref is the same size as (or maybe bigger than) the int, so might as well just copy it?
```suggestion
        const auto x = static_cast<int>(position.X);
        const auto y = static_cast<int>(position.Y);
```